functional-programming-spreadsheet/index040.js

```diff
@@ -0,0 +1,18 @@
+const infixToFunction = {};
+
+function add(x, y) {
+  return x + y;
+}
+
+const addVar = add;
+
+/*
+Anonymous functions are functions without names - they are used only once and then forgotten.
+The syntax is the same as for normal functions but without the name:
+```
+function(x) {
```

Comment:
This might imply that the `const greeting` part is also part of the anonymous function.

I'm thinking of splitting this into two lessons: one to define the anonymous function (which is immediately discarded), and one to assign it to a variable.

Reply:
That's a good point. Good call on splitting it up into two separate lessons.